website/content/en/status/report-2023-07-2023-09/bsd-cafe.adoc

```diff
@@ -1,38 +1,41 @@
+The server operates under clearly defined guidelines that promote positive conduct and unequivocally prohibit any form of hate.
+Inclusiveness, respect, and constructive dialogue stand as the cornerstones of this instance.
 
 Our primary server is currently hosted in Finland on a small VM, running on FreeBSD.
-Our services are partitioned into VNET jails, interconnected within a local area network through a dedicated bridge.
-Additionally, we have implemented a VPN system and have the flexibility to migrate individual jails to more robust machines.
+Services are partitioned into VNET jails, interconnected within a local area network through a dedicated bridge.
+Additionally, we implement a VPN system and have the flexibility to migrate individual jails to more robust machines.
 
-For multimedia data and cache hosting, we have employed a separate physical server (also FreeBSD-based, within a jail), fronted by Cloudflare.
+For multimedia data and cache hosting, we employ a separate physical server (also FreeBSD-based, within a jail), fronted by Cloudflare.
 The goal here is to cache and geodistribute data, effectively reducing network congestion on the main VPS.
 
-Our reverse proxy (frontend), mail server, media server, and the instance itself are all accessible via ipv6.
+Our reverse proxy (frontend), mail server, media server, and the instance itself are all accessible via IPv6.
 
-From the moment of its inception, this instance has remained devoid of preloaded content.
-Our intention is for it to grow organically based on the interests and followers of its users.
+At its inception, this Mastodon instance was devoid of preloaded content.
+Our intention is for it to grow organically, based on the interests and followers of its users.
 At this stage, we have refrained from preemptive blocks.
 We strongly encourage users to promptly report anything that they believe requires attention.
 
-We invite you to join us at https://mastodon.bsd.cafe in order to cultivate a community that values constructive interactions and embraces inclusiveness a secure and serene space meant for all.
+We invite you to join us at https://mastodon.bsd.cafe/[] in order to cultivate a community that values constructive interactions and embraces inclusiveness -- a secure and serene space meant for all.
 
-Furthermore, we have established a website at https://wiki.bsd.cafe, which will provide an overview of our tools, services, rules, uptime, and more.
+Furthermore, we have established a website at https://wiki.bsd.cafe/[], which will provide an overview of our tools, services, rules, uptime, and more.
 
-Recently, a Miniflux installation has been performed, so the BSD Cafe users can use it as a personal RSS Feed Reader. More information can be found here: https://wiki.bsd.cafe/miniflux-bsd-cafe
+Recently, a Miniflux installation has been performed, so the BSD Cafe users can use it as a personal RSS Feed Reader.
+More information: https://wiki.bsd.cafe/miniflux-bsd-cafe[].
```

Comment:
I think we can remove this line: the link has already been given enough times, in particular it is listed in the Links section.

If it is to be kept, the `link` macro should be called explicitly:
```suggestion
More information: link:https://wiki.bsd.cafe/miniflux-bsd-cafe[].
```